cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -282,6 +307,8 @@ func (c bundleCmd) generateMetadata(cfg *config.Config, manifestsDir, outputDir
 	return nil
 }
 
+// NB(estroz): these updates need to be atomic because the bundle's Dockerfile and annotations.yaml
```

Comment:
could you please use Todo since it allows us to find that after.

Reply:
I'll do that for this one because it contains "need". `NB` means non-blocking thing, i.e. "this thing is nice to have but not necessary".